StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory.swift

```diff
@@ -502,6 +504,12 @@ extension PaymentSheetFormFactory {
         )
     }
 
+    func makeExternalPayPal() -> PaymentMethodElement {
+        let contactInfoSection = makeContactInformationSection(nameRequiredByPaymentMethod: false, emailRequiredByPaymentMethod: false, phoneRequiredByPaymentMethod: false)
+        let billingDetails = makeBillingAddressSectionIfNecessary(requiredByPaymentMethod: false)
```

Comment:
You need to call `connectBillingDetailsFields` so the country and phone country elements change together.

Reply:
I saw that but didn't think it made sense to keep, at least as it currently works. It only updates the phone field based on the country field.  iirc I made this behavior for Link, where the phone number is collected *after* the country.  

Outside of Link's phone field, though, we collect phone *before* country, so this code almost always has no effect.

Reply:
Interesting, I only added that _because_ it was already done in the address element, I'm okay not using it anymore.